GlobalSuppressions.cs

```diff
@@ -0,0 +1,99 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Major Code Smell", "S125:Sections of code should not be commented out", Justification = "<Pending>", Scope = "member", Target = "~M:Mirror.Tests.NetworkReaderTest.ReadBytesCountTooBigTest")]
```

Comment:
this one should not be suppressed. only the ones inside weaver should be suppressed

Reply:
fixed